### PR TITLE
Product options

### PR DESCRIPTION
**Pull request by VolodymyrTrach**

### CODE REVIEW

File: modules/product/product.graphql.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ description: [Language]!
 mainMaterial: [Language]!
 innerMaterial: [Language]!
 strapLengthInCm: Int!
-images: PrimaryImage!
+images: [PrimaryImage!]
```
Comment (Crash1212) [inline modules/product/product.graphql.js:11]: Should it be an array?

File: modules/product/product.model.js [hunk 2]
```diff
@@ -18,7 +18,7 @@ const productSchema = new mongoose.Schema({
   mainMaterial: [Language],
   innerMaterial: [Language],
   strapLengthInCm: Number,
-  images: PrimaryImage,
+  images: [PrimaryImage],
```
Comment (Crash1212) [inline modules/product/product.model.js:21]: Should it be an array?

File: modules/product/product.graphql.js (hunk 1 above)
Comment (VolodymyrTrach) [reply modules/product/product.graphql.js:11]: ))) Yep, for now.

File: modules/product/product.model.js (hunk 2 above)
Comment (VolodymyrTrach) [reply modules/product/product.model.js:21]: for now

File: modules/product/product.service.js [hunk 3]
```diff
@@ -85,6 +86,14 @@ class ProductsService {
     };
   }
 
+  async updateProduct(id, products) {
+    const product = await Products.findById(id);
```
Comment (Alan18081) [inline modules/product/product.service.js:90]: I think we need this check here too
`if (await this.checkProductExist(data)) {
      throw new Error(PRODUCT_ALREADY_EXIST);
}`

Comment (VolodymyrTrach) [reply modules/product/product.service.js:85]: done

**Pull request merged**